packages/cxl-lumo-styles/scss/loading.css

```diff
@@ -15,16 +15,14 @@ body::before {
   right: 0;
   bottom: 0;
   left: 0;
-  display: flex;
-  align-items: center;
-  justify-content: center;
```

Comment:
Relative with edge property 0-s automatically puts spinner in middle, really?

Reply:
> Relative with edge property 0-s automatically puts spinner in middle, really?

It was a mistake, rolled back.